test/over_react_test/utils/shadow_nested_component.dart

```diff
@@ -30,7 +30,7 @@ UiFactory<ShadowNestedProps> ShadowNested = uiForwardRef(
 
     useEffect(() {
 	      var shadowRootFirstChild = DivElement()..dataset['test-id'] = props.shadowRootFirstChildTestId ?? 'shadowRootFirstChild';
-	      divRef.current.attachShadow({'mode':'open'}).append(shadowRootFirstChild);
+	      divRef.current?.attachShadow({'mode':'open'}).append(shadowRootFirstChild);
```

Comment:
I feel like we should use `!` here, since if the ref is null, then something's gone wrong with the component, and it isn't doing what we need it to for the tests, and throwing seems more helpful than silently failing.
```suggestion
	      divRef.current!.attachShadow({'mode':'open'}).append(shadowRootFirstChild);
```